### ISSUE TITLE
Accessible label missing of the Send button in SendBox.tsx

### ISSUE THREAD

**Issue by josalomo** [Azure/communication-ui-library#2146]
**Describe the bug; what happened?**
Button to send a chat message is missing discernable text / an accessible label. Button is an interactive element and doesn't have readable text (only an icon SendBoxSend), so a programmatic text / accessible label is needed to convey programmatic meaning, e.g. to screen readers. 

**What are the steps to reproduce the issue?**
Open de chat pane, open Dev Tools and find '<button type="button" class="ms-Button ms-Button--icon root-173" id="sendIconWrapper"', or Accessible Insights > Fast Pass > see the button-name issue. 

**What behavior did you expect?**
The button to have a title or aria-label, so that screen reader users understand the function of the button. 
**If applicable, provide screenshots:**
<img width="337" alt="image" src="https://user-images.githubusercontent.com/63778807/182652237-dca95fdf-db05-45a8-971e-c3ec5392862a.png">
<img width="973" alt="image" src="https://user-images.githubusercontent.com/63778807/182652388-68236578-13d2-4c15-8cd1-9aafd7a40a96.png">

**In what environment did you see the issue?**

- OS & Device: Windows 11 on Surface Laptop 4
- Browser: Microsoft Edge
- Version 103.0.1264.77

**Is there any additional information?**
N.A.

**Comment by JamesBurnside** [comment]
Thanks for filing this @josalomo , we'll take a look as soon as possible and get back to you

**Comment by JamesBurnside** [comment]
HI @josalomo, can you verify you are seeing this issue in the latest version of `@azure/communication-react` (version 1.3.0 or greater).
We recently pushed an accessibility fix in this area that should ensure an aria-label is correctly set #2004

**Comment by josalomo** [comment]
Thanks @JamesBurnside, Think we're indeed using a previous version. Looking at the merged commit of #2004, it indeed looks like an aria-label is present. Will give it a try with version 1.3.0 or newer, and follow-up with the findings.

**Comment by JamesBurnside** [comment]
@josalomo marking this issue as closed. Please leave a comment if you still hit this issue
